firefox-ios/Client/Utils/GleanOhttpUploader.swift

```diff
@@ -47,7 +48,7 @@ struct GleanOhttpUploader: PingUploaderProtocol {
 
     /// Build the request and create upload operation using the `OhttpManager`
     func uploadOhttpRequest(request: GleanPingUploadRequest,
-                            callback: @escaping (UploadResult) -> Void) {
+                            callback: @escaping @Sendable (UploadResult) -> Void) {
```

Comment:
This generates a new warning in `GleanPingUploader`, this will be fixed [with the next Glean update](https://github.com/mozilla/glean/pull/3325)